tests/issue_1512/run_test.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+for jj in {1,2,4,8,16,32,64,128,256,512,1024,2048}; do
```

Comment:
Why the array syntax?  This works just as well and is simpler:
```
$ for i in 1 2 3; do echo $i; done
1
2
3
```

Doesn't matter really - but in general you may want to use POSIX where bash specific features are not needed, to keep scripts portable.